static/js/components/EditFollowupRequestDialog.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import Dialog from "@material-ui/core/Dialog";
+import DialogContent from "@material-ui/core/DialogContent";
+import Form from "@rjsf/material-ui";
+import * as Actions from "../ducks/source";
+
+const EditFollowupRequestDialog = ({
+  followupRequest,
+  instrumentFormParams,
+}) => {
+  const dispatch = useDispatch();
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleSubmit = ({ formData }) => {
+    const json = {
+      allocation_id: followupRequest.allocation.id,
+      obj_id: followupRequest.obj_id,
+      payload: formData,
+    };
+    dispatch(Actions.editFollowupRequest(json, followupRequest.id));
+    handleClose();
+  };
+
+  return (
+    <span key={followupRequest.id}>
+      <button
+        type="button"
+        onClick={handleClickOpen}
+        name={`editRequest_${followupRequest.id}`}
+      >
+        Edit
+      </button>
+      <Dialog open={open} onClose={handleClose} style={{ position: "fixed" }}>
```

Comment:
Minor: use the makeStyles / className approach instead of the inline `style`

Reply:
fixed